### PR TITLE
fix: override dapp gasprice on cfx mainnet

### PR DESCRIPTION
**Pull request by yqrashawn**
PORTAL-2002

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/426)

### CODE REVIEW

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-2002 Flux DEPOSIT cfx操作， 确认Sigh Transaction后，交易一直pending不能发送成功，Scan中看不到pending原因](https://app.clickup.com/t/3719212/PORTAL-2002)

**Review by zctocm — APPROVED** [review]

File: packages/rpcs/cfx_sendTransaction/index.js [hunk 1]
```diff
@@ -66,6 +70,19 @@ export const main = async ({
       throw InvalidParams(`Invalid transaction ${JSON.stringify(params)}`)
     }
 
+    if (app.currentNetwork.name === CFX_MAINNET_NAME) {
+      try {
+        const gasPrice = await cfx_gasPrice({errorFallThrough: true}, [])
```
Comment (angelia-yuqi-personal) [inline packages/rpcs/cfx_sendTransaction/index.js:75]: why use `cfx_gasPrice` not `gasstation_price`
`

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

File: packages/rpcs/cfx_sendTransaction/index.js (hunk 1 above)
Comment (yqrashawn) [reply packages/rpcs/cfx_sendTransaction/index.js:75]: cause it's used in cfx_gasPrice https://github.com/Conflux-Chain/helios/blob/12094ee9e0a054de1fb628a7f22cbaefee6a9c72/packages/rpcs/cfx_gasPrice/index.js#L48

Comment (zctocm) [reply packages/rpcs/cfx_sendTransaction/index.js:75]: `gasstation_price` is in the `cfx_gasPrice` method.

**Review by angelia-yuqi-personal — APPROVED** [review]

**Pull request merged**